static/src/javascripts/projects/common/modules/analytics/omnitureMedia.js

```diff
@@ -13,6 +13,12 @@ define([
             return player.el().getAttribute(attributeName);
         }
 
+        function resetProp71Cookie() {
+            if (config.switches.hostedContentTracking && config.page.toneIds == 'tone/hosted-content') {
+                s.getPreviousValue(s.prop71, 's_prev_ch');
```

Comment:
what does this actually do? and would a test help spec the behaviour in future? i'm not sure if `s_pre_ch` makes sense alone, a comment might help